rest/data/perf_test.go

```diff
@@ -2,6 +2,7 @@ package data
 
 import (
 	"context"
+	"github.com/mongodb/amboy"
 	"testing"
 	"time"
```

Comment:
there's a format-on-save program that sorts imports that we typically use (goimport) that you should be using

Reply:
added goimports and go fmt to apply on save

Reply:
there's something weird... the extra line on 9 shouldn't be here.